README.md

```diff
@@ -63,16 +63,29 @@ Plug 'ThePrimeagen/vim-be-good'
 ### Docker
 
 If you would like, you can use docker to run the game. Doing this will
-automatically use the correct version of neovim for you.
+automatically use the correct version of neovim for you, as well as run the
+game immediately when neovim starts.
+
+#### Stable image
+
+This image always runs the version of the game that was bundled when the image
```

Comment:
I think it would be nice if `This image` was a link to your dockerfile.

Reply:
Excellent point, done. Thanks @FL3SH